nautobot/project-static/js/job_result.js

```diff
@@ -24,7 +24,9 @@ function updatePendingStatusLabel(status) {
 
 function updateLogTable(result_id) {
     // Calls `update_log_table` to refresh the jobs table from the `/log-table/` endpoint
-    update_log_table('', '/extras/job-results/' + result_id + '/log-table/');
+    // Grab the query string from session storage and pass it through to the log table.
+    let qs = window.sessionStorage.getItem("job_results_current_page");
+    update_log_table(qs, '/extras/job-results/' + result_id + '/log-table/');
```

Comment:
The way `update_log_table` works, `null` is overriding the default value of `''` in in it's signature.

```suggestion
    let qs = window.sessionStorage.getItem("job_results_current_page");
    update_log_table((qs || ''), '/extras/job-results/' + result_id + '/log-table/');
```